File: C++/Task06/n37141/n37141.cpp
// https://inf-ege.sdamgia.ru/problem?id=37141

#include <iostream>

using namespace std;

int main()
{
	int max_i = -1001;
	for (int i = -1000; i < 1000; i++)
	{
		int n, s;
		n = 1024;
		s = i;
		while (s >= 5) {
			s = s - 5;
			n = n / 2;
		}
		if (n == 64)
		{
			max_i = i;
		}
	}
	std::cout << max_i << std::endl;
	return 0;
}

File: Python/Task06/n37141.py
# https://inf.reshuege.ru/problem?id=37141

max_i = -1001
for i in range(-1000, 1000):
	n = 1024
	s = i
	while s >= 5:
		s = s - 5
		n = n // 2
	if n == 64:
		max_i = i
print(max_i)